sdk/compiler/damlc/tests/src/DA/Test/DamlcUpgrades.hs

```diff
@@ -1335,5 +438,5 @@ data Expectation
 data Dependency
   = NoDependencies
   | DependOnV1
-  | SeparateDep [(FilePath, String)]
-  deriving (Show, Eq, Ord)
+  | SeparateDep
+  | SeparateDeps
```

Comment:
Nitpick: maybe we should rename this `SeparateDependencies`? Simply worried of the single `s` distinction here is an issue or not?